examples/secrets/put.py

```diff
@@ -0,0 +1,15 @@
+# [START import]
+from nitric.api import Secrets
+# [END import]
+async def secret_put():
+# [START snippet]
+    # Construct a new secret client
+    secrets = Secrets()
+
+    # Store the new password value, making it the latest version
+    new_password = "qxGJp9rWMbYvPEsNFXzukQa!"
+    new_secret = await secrets.secret("my-secret").put(new_password)
+
+    # Access the exact version of the put secret, for future reference
+    new_version = new_secret.secret.version
```

Comment:
The value stored in `new_secret` here _is_ the new version reference, so I think these lines should be changed to:

```python
new_version = await secrets.secret("my-secret").put(new_password)

# Access the exact version of the put secret, for future reference
password = await new_version.access()
```